CONTRIBUTING.md

```diff
@@ -0,0 +1,296 @@
+to it if necessary.
+
+If your Pull Request was accepted, congratulations! You are now an official
+contributor of getkong.org. Your changes will be included in the subsequent release.
+Deploys are currently manually pushed via github pages, so please allow for some
+time for the changes to be persisted to the site.
+
+[Back to TOC](#table-of-contents)
+
+#### Git branches
+
+Everything on `master` will get merged and pushed to github pages. Branch off of
+this branch for local development.
+
+If you have write access to the GitHub repository, please follow the following
+naming scheme when pushing your branch(es):
+
+- `feat/foo-bar` for new features
+- `fix/foo-bar` for bug fixes
+- `tests/foo-bar` when the change concerns only the test suite
+- `refactor/foo-bar` when refactoring code without any behavior change
+- `style/foo-bar` when addressing some style issue
+- `docs/foo-bar` for updates to the README.md, docs, etc.
+
+[Back to TOC](#table-of-contents)
+
+#### Commit atomicity
+
+When submitting patches, it is important that you organize your commits in
+logical units of work. You are free to propose a patch with one or many
+commits, as long as their atomicity is respected. This means that no unrelated
+changes should be included in a commit.
+
+For example: you are writing a patch to fix a bug, but in your endeavour, you
+spot another bug. **Do not fix both bugs in the same commit!**. Finish your
+work on the initial bug, propose your patch, and come back to the second bug
+later on. This is also valid for unrelated style fixes, refactorings, etc...
+
+You should use your best judgement when facing such decisions. A good approach
+for this is to put yourself in the shoes of the person who will review your
+patch: will they understand your changes and reasoning just by reading your
+commit history? Will they find unrelated changes in a particular commit? They
+shouldn't!
+
+Writing meaningful commit messages that follow our commit message format will
+also help you respect this mantra (see the below section).
+
+[Back to TOC](#table-of-contents)
+
+
+#### Commit message format
+
+To maintain a healthy Git history, we ask of you that you write your commit
+messages as follows:
+
+- The tense of your message must be **present**
+- Your message must be prefixed by a type, and a scope
+- The header of your message should not be longer than 50 characters
+- A blank line should be included between the header and the body
+- The body of your message should not contain lines longer than 72 characters
+
+Here is a template of what your commit message should look like:
+
+```
+<type>(<scope>) <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+
+##### Type
+
+The type of your commit indicates what type of change this commit is about. The
+accepted types are:
+
+- **docs**: Changes to the README.md, this file, or static content files associated
+  with Kong CE or EE documentation.
+- **feat**: A new feature
+- **fix**: A bug fix
```

Comment:
let's mention this does not include fixing a typo though. Maybe:

```
- **docs**: Changes made to any static content files associated
  with Kong CE or EE documentation (including typo fixes), the
  README.md or this file.
- **style**: CSS fixes, formatting, missing semi colons, 💅
- **feat**: A new website feature (e.g. JavaScript, HTML, or CSS changes to
  support a new feature.
- **fix**: A website bug fix (related to the JavaScript, HTML, or CSS assets). Typos
  and other fixes to the _contents_ of the documentation (markdown files) are not
  included in this scope. 
``` 

Thoughts? Feel free to rephrase those better for sure ^

Reply:
yea i like that. it's common for people to use `fix` for a typo.